src/pages/[_network]/distributions/getDistributionsDetailsQuery.ts

```diff
@@ -0,0 +1,29 @@
+const getDistributionDetails = (distributionId: string) => `{
+  indexUpdatedEvent(id: "${distributionId}") {
```

Comment:
I want to make this subgraph query through the sfSubgraph object, but I'm not sure how to do that exactly, right now I am querying with fetch to subgraph. I'm sure this can be optimised tho and done internally @kasparkallas

Reply:
Sorry, didn't see this earlier.